apps/api/src/services/data-provider/yahoo-finance/yahoo-finance.service.ts

```diff
@@ -136,6 +138,49 @@ export class YahooFinanceService implements DataProviderInterface {
     }
   }
 
+  public async search(aSymbol: string): Promise<{ items: LookupItem[] }> {
+    let items = [];
+
+    try {
+      const get = bent(
+        `https://query1.finance.yahoo.com/v1/finance/search?q=${aSymbol}&lang=en-US&region=US&quotesCount=8&newsCount=0&enableFuzzyQuery=false&quotesQueryId=tss_match_phrase_query&multiQuoteQueryId=multi_quote_single_token_query&newsQueryId=news_cie_vespa&enableCb=true&enableNavLinks=false&enableEnhancedTrivialQuery=true`,
```

Comment:
are you sure that a user is not able to pass in a custom `aSymbol` which is overwriting other parameters?

Reply:
No. Do you have an idea how to escape the query parameter?

Reply:
I think a normal URL escape should be fine `encodeURIComponent`

Reply:
Okay, I will encode the query in the controller 👍🏻